pkg/tagger/collectors/kubelet_extract_test.go

```diff
@@ -533,22 +533,22 @@ func TestParsePods(t *testing.T) {
 			}},
 		},
 		{
-			desc: "standard container annotations",
+			desc: "standard container labels",
 			pod: &kubelet.Pod{
 				Metadata: kubelet.PodMetadata{
 					Labels: map[string]string{
-						"component":         "kube-proxy",
-						"tier":              "node",
-						"k8s-app":           "kubernetes-dashboard",
-						"pod-template-hash": "490794276",
+						"component":                           "kube-proxy",
+						"tier":                                "node",
+						"k8s-app":                             "kubernetes-dashboard",
+						"pod-template-hash":                   "490794276",
+						"tags.datadoghq.com/dd-agent.env":     "production",
+						"tags.datadoghq.com/dd-agent.service": "dd-agent",
```

Comment:
What happens if the pod is annotated with a different service than the container? Is it additive, so container will get both service tags? Would be good to have a test for that as well.

Reply:
This should not be the most typical use case, but yes, additive. I am adding a test.